Update Community Cloud docs (#1149)
streamlit/docs: content/deploy/community-cloud/manage-your-app/app-settings.md

# App settings

This page is about your app settings on Streamlit Community Cloud. From your app settings you can [view or change your app's URL](/deploy/streamlit-community-cloud/manage-your-app/app-settings#view-or-change-your-apps-url), manage [public or private access to your app](/deploy/streamlit-community-cloud/share-your-app), and update your saved [secrets for your apps](/deploy/streamlit-community-cloud/deploy-your-app/secrets-management).

> If you access "**Settings**" from your [app chrome](/develop/concepts/architecture/app-chrome) in the upper-right corner of your running app, you can access features to control the appearance of your app while its running.

Reviewer: ```suggestion
If you access "**Settings**" from your [app chrome](/develop/concepts/architecture/app-chrome) in the upper-right corner of your running app, you can access features to control the appearance of your app while it's running.
```

### View or change your app's URL

To view or customize your app subdomain from the dashboard:

1. Access your app's settings as described above.
1. On the "**General**" tab in the "App settings" dialog, see your app's unique subdomain in the "App URL" field.

   ![General app settings on Streamlit Community Cloud: Custom subdomain](/images/streamlit-community-cloud/workspace-app-settings-general.png)

> 1. (Optional) Enter a new, custom subdomain between 6 and 63 characters in length, and hit "**Save**".

Reviewer: ```suggestion
1. Optional: Enter a new, custom subdomain between 6 and 63 characters in length, and then click "**Save**."
```
Absolutely NO hitting!

   ![New custom subdomain for your app](/images/streamlit-community-cloud/workspace-app-settings-general-valid-domain.png)

   If a custom subdomain is not available (e.g. because it's already taken or contains restricted words), you'll see an error message. Change your subdomain as indicated.

   ![Invalid custom subdomain for your app](/images/streamlit-community-cloud/workspace-app-settings-general-invalid-domain.png)

### Update your app's share settings

Learn how to [Share your app](/deploy/streamlit-community-cloud/share-your-app).

![Share settings on Streamlit Community Cloud](/images/streamlit-community-cloud/workspace-app-settings-sharing.png)

### View or update your secrets

1. Access your app's settings as described above.
1. On the "**Secrets**" tab in the "App settings" dialog, see your app's secrets in the "Secrets" field.

   ![Secrets app settings on Streamlit Community Cloud](/images/streamlit-community-cloud/workspace-app-settings-secrets.png)

> 1. (Optional) Add, edit, or delete your secrets, and click "**Save**".

Reviewer: ```suggestion
1. (Optional) Add, edit, or delete your secrets, and then click "**Save**."
```
